qa/suites/rbd/migration-external/8-migrate-workloads/execute.yaml

```diff
@@ -0,0 +1,12 @@
+tasks:
+  - parallel:
+      - io_workload
+      - migrate_workload
+migrate_workload:
+  sequential:
+    - exec:
```

Comment:
@orozery can you please cover some migration abort tests here,
so that if any failures during migration user should able to rollback existing source data?

Reply:
There wouldn't be any rollback involved here because the scenario with two sites is supported only in an import-only mode where the source image remains unmodified.  I have tweaked `doc/rbd/rbd-live-migration.rst` to explicitly mention this.